.travis.yml

```diff
@@ -14,6 +15,7 @@ install:
 script:
   - vendor/bin/phpunit src/Tests --coverage-clover=coverage.clover
   - vendor/bin/phpstan analyse
+  - vendor/bin/rector process src --dry-run
```

Comment:
I would let rector to be used by dev one time only, I wouldn't commit it here. Install it globally or do not commit rector stuff